O/OpenBLAS/OpenBLAS@0.3.20/bundled/patches/neoverse-generic-kernels.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/kernel/arm64/KERNEL.NEOVERSEN1 b/kernel/arm64/KERNEL.NEOVERSEN1
```

Comment:
@Keno: https://github.com/xianyi/OpenBLAS/pull/3052 supposedly fixed the issue you reported at https://github.com/xianyi/OpenBLAS/issues/2998, did you ever get a chance to verify whether we can drop this patch now?

Reply:
No idea

Reply:
For the record, I verified the patch should now be redundant, as vanilla OpenBLAS now produces the right result: https://github.com/JuliaPackaging/Yggdrasil/pull/7202#issuecomment-1908397135.